### ISSUE TITLE
A single valid file id should return the result

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons-server#21205]
This has been filed a while ago for users module https://github.com/mozilla/addons-server/issues/20074

Go to files module in admin and make a search using a valid file id 

Expected:
It should return one result 

Actual result: 
0 results found

As a workaround: it is working using 2 ids +

Note:
It cannot be reproduced for users or addons modules.
Reproducible on dev and stage.

**Comment by diox** [comment]
Yeah, `id` or `pk` isn't part of the default search terms we search against in the files admin. To be fair, the tooltip warns about this :)

Need to test but the fix would be to just add `id` to the `search_fields` in `FileAdmin` inside `files/admin.py`.

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDSRV-499
